roles/sensu/server/templates/redis.json.j2

```diff
@@ -0,0 +1,6 @@
+ {
+   "redis": {
+     "port": {{ redis_listen_port }},
+     "host": "{{ redis_interface }}"
```

Comment:
We should not use variables from redis role in case user will want to run redis on separate node or even an instance not deployed by us.